gitax/Visual Studio Projects/C Sharp Projects/GTXLibGit2Sharp/GTXRepo.cs

```diff
@@ -20,8 +20,8 @@ public static class GTXRepo
         /// <param name="repoPath">the repository main path</param>
         public static void Init(string repoPath)
         {
-            if (!Repository.IsValid(repoPath))
-                Repository.Init(repoPath, false);
+            if (!Repository.IsValid(repoPath))            
+             Global.error(repoPath  + " is not a valid git repo");            
```

Comment:
Exactly @ksulli 

Cool @BTXAndy I've been using gitextensions for that. Biased or not, your scenario is actually the #1 issue on this repo that haven't received much attention. Must be implemented and rock solid. Will look into that, right after my azure credit is back